src/components/app.js

```diff
@@ -60,15 +61,26 @@ class App extends React.Component {
           // to check if string value is boolean true
           // eslint-disable-next-line eqeqeq
           if (value === 'true') {
-            query.filenames.push('thumbnail.png')
+            query.filenames.push(THUMB_FILENAME)
           } else {
             // look for 'thumbnail.png' and remove it if we find it
-            const thumbIndex = query.filenames.indexOf('thumbnail.png')
+            const thumbIndex = query.filenames.indexOf(THUMB_FILENAME)
             if (thumbIndex > -1) {
               query.filenames.splice(thumbIndex, 1)
             }
           }
           break
+        case 'hasReadme':
+          if (!Array.isArray(query.filenames)) query.filenames = []
+          // intentionally use double equals type coercion
+          // to check if string value is boolean true
+          // eslint-disable-next-line eqeqeq
+          if (value === 'true') {
```

Comment:
it looks like the comment doesn't match the usage - the comment says `double` equals but the code is `triple` equals.

@hydrosquall could you take a look and decide which approach you like best?

Reply:
Ah, it looks like this was mistaken copy from the previous switch block. When I was testing locally, "strict triple equals" seemed to get the desired result, so I think the fix is to remove this comment from both switch blocks. I'll push up a commit to remove the misleading comment.

Reply:
Addressed in https://github.com/enjalot/blockbuilder-search/pull/96/commits/e9649ae6e99e53bae8573207f9f8a50de2181ff7